pubspec.yaml

```diff
@@ -1,8 +1,10 @@
 name: chewie
 description: A video player for Flutter with Cupertino and Material play controls
-version: 0.8.0
+version: 1.0.0
```

Comment:
What do you think about going to 0.9.0 for now and 1.0.0 in a month or so -- that will allow us to get public usage & feedback before we declare this code stable?

Reply:
The intention was not to declare it stable, but to make it clear that there are incompatible API changes. If we choose to follow semver we have to update the major version number on such breaking changes.

Reply:
Ah, gotcha -- IMO, 1.0.0 releases have that feeling of "ok, we think this is stable now."

And It's totally cool if we change the API before 1.0 according to SemVer in a `0.9.0` release -- pre 1.0 "Anything may change at any time." That, and I've been using the MINOR version (x.MINOR.y) to denote breaking changes on this lib for a bit now :P

Reply:
Ok. But starting from 1.0.0, if we break the API we move to 2.0.0?

Reply:
Yep, exactly. So my thinking: put this at 0.9.0 for the time being. See if we get any more feature requests / bug reports that would require breaking change in the next month or so. If not, we can ship 1.0.0, and if we need to make breaking changes beyond that it'd require a major version bump.